src/libmongoc/tests/test-mongoc-stream.c

```diff
@@ -219,7 +219,7 @@ test_stream_writev_timeout (void)
 {
    bson_error_t error;
 
-   uint8_t data[1] = {0};
+   char data[1] = {0};
    mongoc_iovec_t iov = {.iov_base = data, .iov_len = 1u};
```

Comment:
Good catch. Suggest instead `.iov_base = (void*)data,` rather than `uint8_t -> char` for better consistency with existing usage of `.iov_base`.